common/element_handle_options.go

```diff
@@ -177,6 +191,36 @@ func (o *ElementHandleCheckOptions) Parse(ctx context.Context, opts goja.Value)
 	return o.ElementHandleBasePointerOptions.Parse(ctx, opts)
 }
 
+// NewElementHandleSetInputFilesOptions creates a new ElementHandleSetInputFilesOption.
+func NewElementHandleSetInputFilesOptions(defaultTimeout time.Duration) *ElementHandleSetInputFilesOption {
+	return &ElementHandleSetInputFilesOption{
+		ElementHandleBaseOptions: *NewElementHandleBaseOptions(defaultTimeout),
+		Payload:                  []File{},
+	}
+}
+
+// Parse parses the ElementHandleSetInputFilesOption from the given opts.
+func (o *ElementHandleSetInputFilesOption) Parse(ctx context.Context, opts goja.Value) error {
+	rt := k6ext.Runtime(ctx)
+	if err := o.ElementHandleBaseOptions.Parse(ctx, opts); err != nil {
+		return err
+	}
+	if opts != nil && !goja.IsUndefined(opts) && !goja.IsNull(opts) {
+		opts := opts.ToObject(rt)
+		for _, k := range opts.Keys() {
+			if k == "payload" {
```

Comment:
Instead of nesting, can you also use early-continue here? Like:

`if k != "payload" { continue }`

Reply:
done